src/json_reporter.cc

```diff
@@ -178,6 +178,8 @@ void JSONReporter::PrintRunData(Run const& run) {
     }
     BENCHMARK_UNREACHABLE();
   }()) << ",\n";
+  out << indent << FormatKV("repetitions", run.repetitions) << ",\n";
```

Comment:
@dominichamon @EricWF any better ideas for the name of this field?
(see my previous comments in this PR)

Reply:
`repetitions` is the right name but it shouldn't be reported at the per-run level. i think a repetition index might be the correct way to go without repetitions as you can, in tooling, determine what the max repetition index is for a given run and group them together easily.